### PR TITLE
Tech agnostic contentPlayer

### PR DESCRIPTION
**Pull request by damianvila**
Changed contentPlayer selector to be tech agnostic (will get object
irrespective of it being an html5_api or Flash_api instance). Adds
compatibility for IE 11 browsers using Flash and HLS.

### CODE REVIEW

**Comment by shawnbuso** [comment]
It looks like you may have pulled a whole bunch of stuff from gh-pages into master - can you revert everything except for the changes to videojs.ima.js? Thanks!

File: src/videojs.ima.js [hunk 1]
```diff
@@ -464,7 +464,7 @@
       this.allAdsCompleted = true;
       this.adContainerDiv.style.display = 'none';
       if (this.contentComplete == true) {
-        if (this.contentPlayer.src != this.contentSource) {
```
Comment (shawnbuso) [inline src/videojs.ima.js]: nit: remove trailing spaces

Comment (shawnbuso) [reply src/videojs.ima.js]: Whoops, my mistake, you did remove trailing spaces :)

**Comment by damianvila** [comment]
Yes, sorry for that. Don't know what my GitHub program did (and noticed it too late).

**Comment by damianvila** [comment]
I hope it's ok now. Sorry for the mess...

**Comment by shawnbuso** [comment]
No worries :) I'm not sure why we have that massive list of commits but I can squash them before pulling so that's no problem. It looks like there may be a problem with your CLA though (check out the comment by googlebot above).

**Comment by damianvila** [comment]
I'm not sure if the problem was that I had no public email set. I'm using the same account for GH and the CLA.

**Comment by shawnbuso** [comment]
Huh weird, yea I see your email on the commit and our internal tool says we're good to accept PRs from your GitHub account, so I guess it's an issue with the bot. Thanks!

**Pull request merged**